modules/kubernetes/node-local-dns/charts/node-local-dns/templates/nodelocaldns.yaml

```diff
@@ -86,7 +86,10 @@ data:
         }
     .:53 {
         errors
-        cache 30
+        cache {
+                prefetch 10 1m 25%
+                serve_stale 10m
```

Comment:
After some discussions we came to the conclusion that we should make this configuration evidence based. So lets start off with the default configuration for `serve_stale` and set only `prefetch 10`.

After running this for a while we can through metrics look at if we are for example serving very old stale DNS records or if it is a non issue. Same goes for prefetching.